example/example.dart

```diff
@@ -9,6 +9,27 @@ import 'package:source_gen/generators/json_literal.dart';
 
 part 'example.g.dart';
 
+void main() {
```

Comment:
it might be nice to make this example interactive, I left this test code in but feel free to remove it..